app/assets/stylesheets/modules/success-error.css

```diff
@@ -0,0 +1,36 @@
+.success-error {
+  width: 100px;
+  display: block;
+  margin: 40px auto 0; }
+
+.path {
+  stroke-dasharray: 1000;
+  stroke-dashoffset: 0;
+
+  &.circle {
+    -webkit-animation: dash .9s ease-in-out;
+    animation: dash .9s ease-in-out;
+  }
+
+  &.line {
+    stroke-dashoffset: 1000;
+    -webkit-animation: dash .9s .35s ease-in-out forwards;
```

Comment:
I think we can drop the `webkit` prefix since there's solid [browser support](https://caniuse.com/?search=animation) for `animation` these days.